code/modules/modular_computers/computers/subtypes/dev_silicon.dm

```diff
@@ -0,0 +1,37 @@
+/obj/item/modular_computer/silicon
+	name = "internal computing device"
+	desc = "A synthetic computer."
+	hardware_flag = PROGRAM_SILICON
+	icon_state_unpowered = "laptop-open"
+	icon = 'icons/obj/modular_laptop.dmi'
+	icon_state = "laptop-open"
+	icon_state_broken = "laptop-broken"
```

Comment:
I think these are redundant variables.

Reply:
No. They define icon of computer HUD 'button'.